movies-app/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -53,11 +54,11 @@ export default function ShowContainer() {
   const [reviewsList, setReviewsList] = useState(mockReviewItems);
   const { data, error, isLoading } = useSWR(`/api/shows/${params.id}`, ()=> getShow(params.id as string));
 
-  const { data: reviewsData, error: reviewsError, isLoading: reviewsIsLoading } = useSWR(`/api/shows/${params.id}/reviews`, ()=> getShowReviews(params.id as string));
+  const { data: reviewsData, error: reviewsError, isLoading: reviewsIsLoading } = useSWR(swrKeys.showReviews(params.id as string), ()=> getShowReviews(params.id as string));

```

Comment:
Now that you have this, why are you also keeping the state? You are duplicating data to two states - SWR and useState. No need to do that.

Another question is why do you need reviews in this component? You no longer need to do it with useState and yo no longer need to calculate the average rating.

Reply:
Changed.